### PR TITLE
Updated Constants

### PR DESCRIPTION
**Pull request by hvaranasi9**
I updated the Constants.java file to have the constants for VictorSPX port, the port for the XBox Controller, and a sample constant for kIntakePower.

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -12,4 +12,8 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    public static int kIntakePower = 5;
```
Comment (Piphi5) [inline src/main/java/frc/robot/Constants.java:16]: Intake Power is between 0 - 1, so this needs to be changed
```suggestion
    public static double kIntakePower = something;
```

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -12,4 +12,8 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
```
Comment (Piphi5) [inline src/main/java/frc/robot/Constants.java:15]: To make constants more organized, you can also embed specific constants in a subconstants class.
For example:

```
public static final class IntakeConstants {
// Add relevant intake constants here
}
```

**Review by Piphi5 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Constants.java (hunk 2 above)
Comment (hvaranasi9) [reply src/main/java/frc/robot/Constants.java:15]: Thank you for the suggestion Matteo! I have made changes accordingly.

File: src/main/java/frc/robot/Constants.java [hunk 3]
```diff
@@ -12,4 +12,10 @@
...
+    public static final class IntakeConstants {
+        public static double kIntakePower = something;
```
Comment (Piphi5) [inline src/main/java/frc/robot/Constants.java]: Set this to a double

File: src/main/java/frc/robot/Constants.java [hunk 4]
```diff
@@ -12,4 +12,10 @@
...
+    }
+    public static final int VictorSPXPort = 2;
```
Comment (Piphi5) [inline src/main/java/frc/robot/Constants.java:17]: I would recommend including this in the IntakeConstants

**Review by Piphi5 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Constants.java (hunk 4 above)
Comment (hvaranasi9) [reply src/main/java/frc/robot/Constants.java:17]: Should I also include the VictorSPXPort in IntakeConstants?

Comment (Piphi5) [reply src/main/java/frc/robot/Constants.java:17]: Yes

Comment (hvaranasi9) [reply src/main/java/frc/robot/Constants.java:17]: Ok, I made this change as well as other suggested changes, may you please take a look when you get a chance?

Comment (Piphi5) [reply src/main/java/frc/robot/Constants.java:17]: Looks good

**Review by Piphi5 — APPROVED** [review]

**Comment by Piphi5** [comment]
I would recommend that you use an [interactive rebase](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) and squash everything together so that this is just one commit.

Do note that you need to do a 
`git push origin feature --force`
to push the rebased changes onto your branch

File: src/main/java/frc/robot/Constants.java (hunk 4 above)
Comment (hvaranasi9) [reply src/main/java/frc/robot/Constants.java:17]: Alright, thank you!

**Pull request merged**